repose-aggregator/components/filters/openstack-identity-v3/src/test/scala/org/openrepose/filters/openstackidentityv3/OpenStackIdentityV3HandlerTest.scala

```diff
@@ -3,12 +3,13 @@ package org.openrepose.filters.openstackidentityv3
 import java.util
 
 import com.mockrunner.mock.web.{MockHttpServletRequest, MockHttpServletResponse}
-import org.openrepose.core.filter.logic.{FilterAction, FilterDirector, HeaderManager}
 import org.junit.runner.RunWith
+import org.mockito.Matchers.{eq => mockitoEq}
```

Comment:
reason #205 why scala is kind of awesome: aliasing of function names in the import